index.bs

```diff
@@ -5253,10 +5266,29 @@ as taking a Boolean client argument, set to [TRUE] to signify that the extension
 
 Extensions that only affect client processing need not specify [=authenticator extension input=]. Extensions that have
 authenticator processing MUST specify the method of computing the [=authenticator extension input=] from the [=client extension
-input=]. For extensions that do not require input parameters and are defined as taking a Boolean [=client extension input=]
-value set to [TRUE], this method SHOULD consist of passing an [=authenticator extension input=] value of [TRUE] (CBOR major type
+input=],
+and MUST define extensions for the [=CDDL=] types
+<code>[[#iface-authentication-extensions-authenticator-inputs|extensionInputs]]</code>
+and <code>[[#iface-authentication-extensions-authenticator-outputs|extensionOutputs]]</code>
+by defining an additional choice for the `$$extensionInput` and `$$extensionOutput` groups.
```

Comment:
maybe:
```suggestion
by defining an additional choice for the `$$extensionInput` and `$$extensionOutput` [=group sockets=].
```
add to anchors up at top of index.bs: 
```
spec: RFC8610; urlPrefix: https://tools.ietf.org/html/rfc8610
    type: dfn
        text: group sockets; url: section-3.9
```

Reply:
Good idea, thanks!